test/tests/jobs/GetJobTest.cpp

```diff
@@ -1,5 +1,25 @@
 #include <test/lib/BedrockTester.h>
 
+bool isBetweenSecondsInclusive(uint64_t startTimestamp, uint64_t endTimestamp, string timestampString) {
+    uint64_t testTime = startTimestamp;
+    while (true) {
+        string testTimeString = SComposeTime("%Y-%m-%d %H:%M:%S", testTime);
+        if (timestampString == testTimeString) {
```

Comment:
move these lines out of the loop

Reply:
Can't, `testTime` changes throughout the loop, so `testTimeString` changes as well.